src/PowerPlaywright/Pages/EntityRecordPage.cs

```diff
@@ -22,5 +22,8 @@ public EntityRecordPage(IPage page, IControlFactory controlFactory)
 
         /// <inheritdoc/>
         public IMainForm Form => this.GetControl<IMainForm>();
+
+        /// <inheritdoc/>
+        public ICommandBar CommandBar => this.GetControl<ICommandBar>();
```

Comment:
Thank you for sharing this MR! I’m still getting up to speed with the framework’s architecture, so please forgive any basic questions. Since we’ll be adding a CommandBar to EntityRecord page, should we also include the same CommandBar property on IEntityListPage? 

![image](https://github.com/user-attachments/assets/2b9e82ad-78f4-4ebb-a85c-b0e0cf93eff3)

is that means we need to add same CommandBar property to IEntlityListpage as well?